community-guidelines.md

```diff
@@ -0,0 +1,51 @@
+# Polyhack Community Guidelines
+
+**[#polyhack](https://github.com/polyhack)** is a community of
+programmers based on the IRC freenode network. It is specifically a
+polyglot group concerned to discuss programming ideas over the long-term.
+
+Its membership is predominantly from the Australian east coast but
+includes members from North America, New Zealand, and further. It was
+created in August, 2012.
```

Comment:
Again, this is straight from the homepage. Now that I think about it, this paragraph doesn't really add anything that is relevant to the topic, so I will remove it from here.